.github/workflows/check_release_name_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Check release bump
+
+env:
+  TZ: Europe/London
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: get release-version
+      run: |
+        echo "RELEASE_VERSION=$(printf '%s\n' $(cat .release-version))" >> $GITHUB_ENV
```

Comment:
Does this deal with newlines at the end of the .release-version file? Many IDEs will add a line when saving so we will need to check that.

Reply:
tested this locally - adding new lines and running this command in bash and any new lines added were removed